cmd/spire-server/cli/entry/create_test.go

```diff
@@ -289,6 +289,7 @@ StoreSvid        : true
       },
       "entry": {
         "id": "entry-id",
+        "hint": "",
```

Comment:
why hint is required here? looks like it is not related with this PR (the same comment is for all files with the same change)

Reply:
The new version of spire-api-sdk comes with the hint field change; that's why I had to include those here. This field will be effectively included in https://github.com/spiffe/spire/pull/3926